### DISCUSSION TITLE
栅格布局如何隐藏某列？

### DISCUSSION THREAD

**Discussion by snowiceyeah** [HandyOrg/HandyControl#1105] [Q&A] [answered]
栅格布局当col可见性（Visibility）设置为Collapsed时，如何控制其他控件自动往前顶？
![image](https://user-images.githubusercontent.com/43429385/149869335-0aa16a6f-c747-465a-99ae-fe9d51e08d43.png)
![image](https://user-images.githubusercontent.com/43429385/149869364-bcba6da5-9ed2-4b91-8ad6-daada03b1a43.png)

**Comment by NaBian** [comment, marked as answer]
closed in 31501d1f
